pkg/startup/startup.go

```diff
@@ -164,39 +165,16 @@ func Run() {
 			} else {
 				log.WithError(err).Error("failed to query Rancher's cluster state config map")
 				terminate()
-				return
 			}
 		}
 	}
 
-	// Configure and verify the node IP addresses and subnets.
```

Comment:
If we're putting these into helper functions, any reason it needs to be two separate functions rather than just a single function?